templates/careers/jobs/job-detail.html

```diff
@@ -30,16 +30,21 @@ <h1>Careers</h1>
 {% block overview %}
   <div class="row u-sv3">
     <div class="col-12">
-      <a href="#">&lsaquo;&nbsp;Back to list</a>
+      <a href="{{ request.referrer }}">&lsaquo;&nbsp;Back to list</a>
     </div>
   </div>
   <div class="row">
     <div class="col-12">
-      <h2 class="u-no-margin--bottom u-sv1">Robotics Developer Advocate&nbsp;›</h2>
-      <p class="p-muted-heading">Anywhere - Home based</p>
+      <h2 class="u-no-margin--bottom u-sv1">{{ job.title }}</h2>
+      <p class="p-muted-heading">{{ job.location }}</p>
     </div>
   </div>
   <div class="row">
+    <div class="col-12">
+      {{ job.content | safe }}
+    </div>
+  </div>
+  <!-- <div class="row">
```

Comment:
Is this still here because you're going to uncomment and implement it in a future PR?

In general, we shouldn't leave commented-out code hanging around, 'cos it'll always be in version control history anyway if we really need it.

Reply:
Yes, the idea was to use some different type of design for job details, however this needs to be discussed further. I will remove it from now as we have it available in version control history, as you said.